python/cugraph/cugraph/structure/property_graph.py

```diff
@@ -1326,6 +1334,11 @@ def __get_all_vertices_series(self):
         if epd is not None:
             vert_sers.append(epd[self.src_col_name])
             vert_sers.append(epd[self.dst_col_name])
+        if len(vert_sers) > 1 and not all(
```

Comment:
Can you also update the docstring for this method to mention that the index for the returned Series objects will be normalized/reset if the dtypes differ?

Reply:
This code has been deleted. The test allowed me to confirm that it's not needed (unlike for MG, where it is).  `concat` with `ignore_index=True` works as expected.

Also, I don't think the indices here are meaningful.